packages/terra-clinical-item-view/src/terra-dev-site/doc/clinical-item-view/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,230 @@
+Alternatively, when we use the trueColumn layout and there is a display on one side with overflow, no blank lines are added below the display in the opposite column. The next display is rendered directly under the previous one with no blank lines.
+
+All of that being said, the screenreader readout for the 'by row' formatting announces the first display from the first column and then jumps over to the first display in the second column.
+It then jumps back to the first column and reads out the next display before moving back over to the second column and reading the next display. This pattern of reading displays by row continues until the last display is read.
+
+<Notice variant="important" ariaLevel="3">
+
+The primary reason this 'by row' formatting is not recommended is because visually when information is shown in a column it is natural to read all of the information in the first column and then the second.
+Non-sighted users should also have the opportunity to take in information this way.
+
+</Notice>
+
+How these views are used, what kind of information is being communicated, and how far apart the columns are - all of these contribute to how understandable the information is to both non-sighted and sighted users.
+**It is always best practice in general to keep closely related information in the same column.**
+
+#### Styling Override
+
+Clincial Item View takes in Clinical Item Displays and by default the Item View has its own styling for these display elements.
+However, the Item Displays have several different types of styling that can be applied to each individual display. Sometimes this styling conveys important information, like a display with a strikethrough.
+
+To allow the usage of custom styling we have the `overrideDefaultStyling` prop.
+
+This prop allows the user to override the Item View default styling and instead use the styling directly applied to the Item Displays.
+If the override prop is set to true but the displays have no specific styling set, the displays will end up with the default Item Display styling.
+More information on these styling types can be found on the Clincial Item Display [**page**](https://engineering.cerner.com/terra-clinical/components/terra-clinical-item-display/clinical-item-display/clinical-item-display).
+
+#### Truncation Handling
+
+There is an `isTruncated` prop for Clinical Item View, when this is set to true all displays and comments in the Item View can be truncated. Otherwise, the content wraps to a newline when truncation is false.
+The actual Item Displays themselves also have an `isTruncated` prop. If the Item View has truncation set to false but some of the Item Displays have it set to true, only those specific displays will be truncated. The rest will have the word wrap behavior.
+
+<Notice variant="important" ariaLevel="3">
```

Comment:
Same extended use of Notice here. We make this a proper section and limit the Notice to:
<img width="1016" alt="CleanShot 2023-09-21 at 17 21 29@2x" src="https://github.com/cerner/terra-clinical/assets/38024451/d39fcd5b-4879-4084-8ff9-8b976c823f88">

Reply:
So this one in particular I pulled from Item Display: https://github.com/cerner/terra-clinical/blob/main/packages/terra-clinical-item-display/src/terra-dev-site/doc/clinical-item-display/clinicalItemDisplay.1.doc.mdx#L47-L71

It's also seen on their main site page: https://engineering.cerner.com/terra-clinical/components/terra-clinical-item-display/clinical-item-display/clinical-item-display

Should I still go ahead and change this?

Reply:
Oh. In that case, let's keep it for now and @mjpalazzo & I can review the information/usage of Notice there.